composer.json

```diff
@@ -32,7 +32,7 @@
         "doctrine/mongodb-odm": "^1.3.0 || ^2.0.0",
         "doctrine/orm": "^2.14 || ^3",
         "fig/log-test": "^1",
-        "phpstan/phpstan": "1.12.21",
+        "phpstan/phpstan": "1.12.21 || 2.1.8",
```

Comment:
```suggestion
        "phpstan/phpstan": "2.1.8",
```

🤔

Reply:
Oh I had no idea 7.4 was supported, that's great!